### DISCUSSION TITLE
windows compability

### DISCUSSION THREAD

**Discussion by AliBilalSindhi** [beeb/pancaketrade#61] [Q&A]
kindly tell me does it work on windows laptop or I have to download LinuxOS

**Comment by beeb** [comment]
I would strongly advise against running the script from a laptop as it is really meant to be running continuously without interruption. I have not tested installing the dependencies on Windows. You'll have to try for yourself and I cannot help in case it doesn't work.

**Reply by beeb** [reply]
You might consider using docker if you can only use windows.

**Reply by AliBilalSindhi** [reply]
Thanks

**Reply by Odeneus** [reply]
Hi when you say continuously running without interruption, you mean when you have active trades on the bot right? i mean I assume you can trun off laptop when you are not using the bot, or will that mess up the program or something?

**Reply by beeb** [reply]
You cannot turn off the machine that runs the script. It has to run (process is alive) continuously, in order to track the prices and trigger the orders at the right moment.

**Reply by Odeneus** [reply]
Hi thanks for your reply. So I understand that the machine needs to stay on in order for the script/bot to be able to work, but i mean if i want to take a break from trading and don't have any active trades, it's ok to turn it off and on again right? This won't mess up the script or harm it in any way?

**Reply by beeb** [reply]
Yes this is correct, information is persisted in a file database and the script can be stopped or restarted at any time. Any price tracking will not happen during down time, but some orders might trigger as soon as the script is started again if the conditions are met. I suggest to delete all orders before stopping the script for a longer period of time.

**Reply by Odeneus** [reply]
ok thanks for your reply, i fully understand it now
